xem

```diff
@@ -20,6 +20,7 @@ xem is a Scala program wrapped in a shell script. It takes the source files dire
         --mask-elements             [a comma-separated list of XML element tags, whose content and sub elements should be masked with CDATA]
         --target-files-directory    [ABSOLUTE PATH TO TARGET FILES DIRECTORY]
         --content-only              utilise only the content of the sub elements (i.e. without further XML elements tags inside)
+        --exclude-empty-elements    exclude empty XML element tags, which should be masked with CDATA
```

Comment:
ok, I assumed that this was the cause that produced the invalid XML ;)
so using this option will remove these XML elements overall (?), i.e., you can't then even make use of an exists function later (which might be a disadvantage)